### ISSUE TITLE
`MaxLineLength` and raw strings

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#5060]
## Expected Behavior
```kotlin
"""
    MaxLineLength shouldn't raise any finding when it is find inside a raw string
"""
```

## Observed Behavior
It does: https://github.com/detekt/detekt/pull/5059#discussion_r917284016

## Context
#5059

## Your Environment
* Version of detekt used: main / 1.21.0-RC2
* Gradle scan link: https://ge.detekt.dev/s/2lpijultktfe2

**Comment by BraisGabin** [comment]
Go for it
